man/notin.Rd

```diff
@@ -0,0 +1,34 @@
+\name{notin}
+\alias{\%notin\%}
+
+\title{
+Convenience for checking if an example is in a set of elements
+}
+
+\description{
+Intended to behave opposite to  \code{\link[=base]{in}}
```

Comment:
"Opposite" doesn't feel very precise here (opposite how?), how about "Check whether an object is absent from a table, i.e., the logical inverse of `%in%`"?